index.html

```diff
@@ -7,57 +7,56 @@
- </head>
-
-<body>
+  </head>
+  <body>
     <div class="container-header"> 
-        <header class="masthead">
-            <h1 class="masthead-name">Daniel Affourtit</h1>
-            <h2 class="masthead-slogan">Instructional Design Meets Web Development</h2>
-        </header>
-    </div>
-    <div>   
-        <section class="intro">
-          <h2>a Modern <span>Pedagogue</span> with a dynamic mix of</h2>
-          <h2><span>Instructor</span>, <span>Instructional Designer</span>, and <span>Web Developer</span></h2>
-        </section>
+      <header class="masthead">
+        <h1 class="masthead-name">Daniel Affourtit</h1>
+        <h2 class="masthead-slogan">Instructional Design Meets Web Development</h2>
+      </header>
     </div>
+    <section class="intro">
+      <h2>a Modern <span>Pedagogue</span> with a dynamic mix of</h2>
+      <h2>
+        <span>Instructor</span>, <span>Instructional Designer</span>, and <span>Web Developer</span>
+      </h2>
+    </section>
     <div class="container-content">
       <section class="portfolio">
-          <h2>my portfolio</h2>
-            <article class="group">
-              <h3>Work #1</h3>
-                 <a href="#">
-                   <figure>
-                   <img src="http://i840.photobucket.com/albums/zz326/daf4tee/pptExample.png" > 
-                  </figure>
-                </a>
-            </article>
-             <article class="group">
-              <h3>Work #2</h3>
-                  <figure>
-                    <img src="http://i840.photobucket.com/albums/zz326/daf4tee/preziExample.png" />
-                  </figure>
-            </article>
-            <article class="group">
-              <h3>Work #3</h3>
-                <a href="#">
-                   <figure>
-                    <img src="http://i840.photobucket.com/albums/zz326/daf4tee/psConf2014.png" />
-                  </figure>
-                </a>
-              </article>
-        </section>
+        <h2>my portfolio</h2>
+        <article class="group">
+          <h3>
+            Work #1
+          </h3>
+          <a href="#">
+            <figure>
+              <img src="http://i840.photobucket.com/albums/zz326/daf4tee/pptExample.png" > 
+            </figure>
+          </a>
+        </article>
+        <article class="group">
+          <h3>Work #2</h3>
+          <figure>
+            <img src="http://i840.photobucket.com/albums/zz326/daf4tee/preziExample.png" />
+          </figure>
+        </article>
+        <article class="group">
+          <h3>Work #3</h3>
+          <a href="#">
+            <figure>
+              <img src="http://i840.photobucket.com/albums/zz326/daf4tee/psConf2014.png" />
+            </figure>
+          </a>
+        </article>
+      </section>
     </div>
     <div class="skills">
-        <section>
-            
-        </section>
+      <section>
+        &nbsp;
```

Comment:
```&nbsp;``` is a blank character code and is good to use as a placeholder in elements that don't have any content yet so that the browser renders the element normally with the proper parameters for the box-model & font-size, line-height, etc.